imageplugin/external_image_manager.go

```diff
@@ -93,6 +93,7 @@ func (p *ExternalImageManager) Destroy(log lager.Logger, handle, rootfs string)
 	log.Debug("start")
 	defer log.Debug("end")
 
+	rootfs = strings.TrimSuffix(rootfs, "/rootfs")
```

Comment:
technically probably better to use one of the methods from `filepath` rather than string manipulation here?